operations/README.md

```diff
@@ -0,0 +1,9 @@
+# TOC Operations
+
+This directory contains various files, procedures, responsibilties, and expectations of TOC members. Within this folder, TOC members and the community can understand more about how the TOC conducts themselves, what work they perform and how. As the TOC continues to refine and improve its processes, this directory will be updated.
+
+## Operations structure
+
+* [TOC Onboarding](onboarding.md) - a collection of materials, general information, and other relevant details to support new and returning TOC members in getting up to speed.
+* [Decision Process](toc-decision-process.md) - General information on how we make decisions.
+* [TOC Chair](toc-chair.md) - Additional onboarding, responsibilities, expectations of the TOC Chair.
```

Comment:
Should we also have an expectation for TOC member?

Reply:
that should be included in the onboarding document, i can update the bullet at line 7 to reference expectations. PTAL and let me know if the contents in onboarding convey expectations or if we need additional material.

Reply:
Yep, the on-boarding doc (particularly the Attendence, Availability, and Involvement section) shows minimum requirement for TOC member, which seems much smaller compared with responsibilities of chairs. :)

Reply:
+1 on having a bullet for TOC member expection to link to that section.